run_scripts/1_coda_get.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# -ne 3 ]]; then
+    echo "Usage: ./1_coda_get.sh <coda-auth-file> <coda-v2-root> <data-root>"
+    echo "Downloads coded messages datasets from Coda to '<data-root>/Coded Coda Files'"
+    exit
+fi
+
+AUTH=$1
+CODA_V2_ROOT=$2
+DATA_ROOT=$3
+
+./checkout_coda_v2.sh "$CODA_V2_ROOT"
+
+DATASETS=(
+    "KE_VAX_vaccination_thoughts"
```

Comment:
we need to replace this with the research question datasets for this project.

Reply:
Thanks, updated.